src/pymap/admin.py

```
@@ -0,0 +1,188 @@
+    IntervalSchedule,
+    ClockedSchedule,
+    CrontabSchedule,
+    PeriodicTask,
+)
+from django_celery_beat.admin import (
+    PeriodicTaskAdmin,
+    ClockedScheduleAdmin,
+    CrontabScheduleAdmin,
+)
+
+from migrator.models import CeleryTask, UserPreferences
```

Ruff (0.12.2)

`migrator.models.CeleryTask` imported but unused

Remove unused import

(F401)